src/commands/init.rs

```diff
@@ -1,15 +1,15 @@
 use std::io::Write;
-use std::path::PathBuf;
 
-use color_eyre::eyre::{eyre, Context, Result};
+use color_eyre::eyre::{Context, Result};
 
 use crate::config::{create_config, ChainConfig, Config, FarmerConfig, NodeConfig};
 use crate::utils::{
-    get_user_input, is_valid_chain, plot_location_getter, print_ascii_art, print_version,
+    chain_parser, get_user_input, node_name_parser, plot_directory_parser, plot_location_getter,
+    print_ascii_art, print_version, reward_address_parser, size_parser,
 };
 
 /// defaults for the user config file
-const DEFAULT_PLOT_SIZE: bytesize::ByteSize = bytesize::ByteSize::gib(100);
+const DEFAULT_PLOT_SIZE: bytesize::ByteSize = bytesize::ByteSize::gb(100);
```

Comment:
I still think that `Gib`s here is better than `Gb`. It is a nice number for a protocol (proper value of sectors without a reminder)

Reply:
i agree. My priority was to have consistency (all `gib` or all `gb` in the codebase). 
My second concern was, if `gib` would be confusing for the users. I mean, `gib` makes more sense for sure. And in `sdk` I support it. In CLI however, this is a user-facing app. Users may prefer to see `gb`
Feel free to correct me at any point. I'm not completely against turning all these into `gib` as well